test/processors/file_processor_test.rb

```diff
@@ -23,9 +32,14 @@ class FileProcessorTest < ActiveSupport::TestCase
     }
 
     it 'should copy file' do
+      if travis?
+        audio_monster.expect(:info_for, { format: 'wav' }, [String])
+        audio_monster.expect(:info_for, { format: 'wav' }, [String])
+        audio_monster.expect(:info_for, { format: 'wav' }, [String])
+      end
+
       processor.on_message(msg)
-      processor.result_details[:info][:file].must_match /WAVE/
+      processor.result_details[:info][:format].must_equal 'wav'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.